src/test/resources/testconfigs/broad.json

```diff
@@ -26,5 +26,7 @@
     }
   ],
   "projectForExternalResources": "terra-cli-test",
-  "useJanitorForExternalResourcesCreatedByTests": true
+  "useJanitor": true,
+  "janitorPubSubTopic": "crljanitor-tools-pubsub-topic",
```

Comment:
it's kinda redundant to have `-pubsub-topic` in a topic name, but I guess this exists already.

Also, is this config for all Broad envs?

Reply:
agreed, but the topic already exists so 🤷 
yes, we plan to use the same Janitor instance (and same pubsub topic) to clean up after all envs, though that's currently just dev.